src/main/java/com/google/sps/servlets/KeywordServlet.java

```diff
@@ -0,0 +1,37 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import com.google.gson.Gson;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Given a tweet or sentence, the servlet will extract salient terms from those
+ * and store them in a list.
+ */
+@WebServlet("/keyword")
+public class KeywordServlet extends HttpServlet {
+
+  private static final List<String> tweets = new ArrayList<>();
```

Comment:
Should we use a list data structure for now if we're starting with one tweet, or should we stick with just a single string?

Reply:
I think for the purposes of the prototype, this would do. Perhaps I'll make it so that the textarea in which you input the tweet is pre-filled with the "hard-coded" tweet.

Reply:
Actually just looked at the new design doc comments and I think I misunderstood, the way you have it is good.